pkg/collect/runner.go

```diff
@@ -349,11 +349,11 @@ func getContainerLogsInternal(ctx context.Context, client kubernetes.Interface,
 
 	err = retry.OnError(retry.DefaultBackoff, retryableFn, logsFn)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	if strings.Contains(string(logs), "Internal Error") {
```

Comment:
Lets also to convert our fictional 1G file to a string. It just duplicates the data to memory. Search through the byte slice.

```suggestion
	if bytes.Contains(logs, []byte("Internal Error")) {
```